src/main/java/ch/bfh/bti7081/view/Layout.java

```diff
@@ -7,7 +7,7 @@
 
 public class Layout extends VerticalLayout implements RouterLayout {
     private final HorizontalLayout menuBar = new HorizontalLayout(
-            new RouterLink("Startseite", MainView.class),
+            new RouterLink("Startseite", MainViewContainer.class),
```

Comment:
Warum das renaming?

Reply:
Bin mit dem Renaming auch nicht glücklich. Entsprach vorher unter anderem auch der Präsentation zu MVP, in erster Linie sollte wir halt darauf achten das alle dieselbe Notation verwenden. 
Die Klasse die View wie auch Presenter linkt sollte m.E. nicht Container heissen, da sie keiner ist.

Reply:
Renaming entspricht dem MVP Pattern gemäss Empfehlung des Dozenten
ist im Calculator-Beispiel ähnlich gelöst (MainviewContainer=Mainview, MainViewPresenter=CalculatorPresenter, MainViewContent=CalculatorViewImpl). Container ist Vorschlag des Dozenten. Hast du eine bessere Idee?